ide/coq.ml

```diff
@@ -9,44 +9,6 @@
 open Ideutils
 open Preferences
 
-(** * Version and date *)
-
-let get_version_date () =
-  let date =
-    if Glib.Utf8.validate Coq_config.date
-    then Coq_config.date
-    else "<date not printable>" in
-  try
-    (* the following makes sense only when running with local layout *)
-    let coqroot = Filename.concat
-      (Filename.dirname Sys.executable_name)
-      Filename.parent_dir_name
-    in
-    let ch = open_in (Filename.concat coqroot "revision") in
-    let ver = input_line ch in
-    let rev = input_line ch in
-    (ver,rev)
-  with _ -> (Coq_config.version,date)
-
-let short_version () =
-  let (ver,date) = get_version_date () in
-  Printf.sprintf "The Coq Proof Assistant, version %s (%s)\n" ver date
-
-let version () =
-  let (ver,date) = get_version_date () in
-    Printf.sprintf
-      "The Coq Proof Assistant, version %s (%s)\
-       \nArchitecture %s running %s operating system\
-       \nGtk version is %s\
```

Comment:
Why are we removing information about the Gtk version?  (Commit message suggests this is dead code.... do we want to put this info back in somewhere?)

Reply:
No opinion on that, @maximedenes ?